src/test/java/de/retest/suite/flow/CreateChangesetForAllDiffrencesFlowTest.java

```diff
@@ -0,0 +1,100 @@
+package de.retest.suite.flow;
+
+import static org.assertj.core.api.Assertions.assertThatCode;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import java.io.File;
+import java.nio.file.Files;
+import java.util.Arrays;
+import java.util.HashSet;
+
+import org.apache.commons.io.FileUtils;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+
+import de.retest.recheck.configuration.ProjectConfiguration;
+import de.retest.recheck.persistence.NoStateFileFoundException;
+import de.retest.recheck.persistence.Persistence;
+import de.retest.recheck.persistence.PersistenceFactory;
+import de.retest.recheck.persistence.xml.util.StdXmlClassesProvider;
+import de.retest.recheck.suite.flow.ApplyChangesToStatesFlow;
+import de.retest.recheck.ui.descriptors.Element;
+import de.retest.recheck.ui.descriptors.SutState;
+import de.retest.recheck.ui.diff.AttributeDifference;
+import de.retest.recheck.ui.review.ReviewResult;
+import de.retest.recheck.ui.review.SuiteChangeSet;
+import de.retest.recheck.util.junit.vintage.SystemProperty;
+
+public class CreateChangesetForAllDiffrencesFlowTest {
+
+	private static final String SUITE = "ApplyChangesToStatesFlowTest";
+	private static final String TEST = "check_GUI_with_review_license";
+	private static final String ACTION = "launch";
+	private static final String FILE = "src/test/resources/recheck/" + SUITE + "/" + TEST + "." + ACTION + ".recheck";
+
+	@Rule
+	public TemporaryFolder temp = new TemporaryFolder();
+
+	@Rule
+	public SystemProperty recheckDirProp = new SystemProperty( ProjectConfiguration.RETEST_PROJECT_ROOT );
+
+	@Before
+	public void setUp() throws Exception {
+		recheckDirProp.setValue( temp.getRoot().getAbsolutePath().toString() );
+		Files.createDirectories( temp.getRoot().toPath().resolve( "src/test/java" ) );
+	}
+
+	@Test
+	public void apply_should_throw_exception_whitout_the_stateFilePath() throws Exception {
+		final PersistenceFactory persistenceFactory =
+				new PersistenceFactory( new HashSet<>( Arrays.asList( StdXmlClassesProvider.getXmlDataClasses() ) ) );
+
+		final Persistence<SutState> persistence = persistenceFactory.getPersistence();
+		final File originalFile = new File( FILE );
```

Comment:
Why not move the entire file copying and so forth into `setup()`?

Reply:
Done